### ISSUE TITLE
Unused declarations analyzer not working

### ISSUE THREAD

**Issue by BentTranberg** [dotnet/fsharp#6706]
Unused open statements and declarations are no longer grayed out. I don't remember when I last had it, but I miss it very much. Problem on two different machines. I'm at Visual Studio 2019 v 16.0.3.

Yes, I did research whether there was another open issue, some news I missed, some setting that perhaps was changed, including coloring, but I can't see anything wrong. If it's coloring that's bad, I am not sure which color to reconfigure, but I don't think it's coloring.

Related feature issue:
https://github.com/microsoft/visualfsharp/issues/2354

**Comment by cartermp** [comment]
@BentTranberg It triggers for me in a few F# projects I have on my machine. Do you have a repro that can be shared?

**Comment by BentTranberg** [comment]
No, this is not related to a specific solution or project. I can open anything F#, however old, and there are no grayouts.

**Comment by BentTranberg** [comment]
I checked out an older solution with C# and F#, and I have the same problem in C#.

By checking out fresh, I leave out old .vs, bin, obj, .user, .suo, and all kinds of things that I suspect might keep troublesome settings related to this. There's only .sln, .fsproj, .csproj, and plain C# and F# source files.

In Tools/Options/Text Editor/C#/Advanced there is group Fading with **Fade out unused usings** and **Fade out unreachable code**, both checked, and neither works. I double checked by mousing over C# code and it says "Unreachable code detected" but is still not faded.

I carefully read through the list of colors again. I can't imagine the trouble is with one of them, but can't be sure.

It won't work with new projects either on my machines, in case you wonder about that. Not sure if it makes sense to post a new solution as a repro. Maybe with the .vs folder and all other stuff included?

**Comment by BentTranberg** [comment]
There is a new feature in VS, as of v 16.0.4 I believe, or perhaps .3. This is a screenshot :

![image](https://user-images.githubusercontent.com/15068882/57829750-46f63200-77b0-11e9-9891-7044b1a28265.png)

I'm referring to that grayed out commentlike thing. Not sure what the feature is called. Very nice by the way.

Is it possible that any work leading up to this feature is related to the vanishing graying out (fading) of opens and declarations that I experience?

**Comment by cartermp** [comment]
@BentTranberg No, it's a separate thing altogether. The fact that you're not seeing things grey out with C# either leads me to believe you might have a corrupted VS install, assuming you don't have things turned off.

**Comment by BentTranberg** [comment]
Right. Everything is very much the same on the two machines affected, so it's quite possible the same problem arose on both in parallel. If other people were affected by this, I should have discovered it by now through googling or responses here. Since apparently not, **I suggest we close this one now, to keep the noise level down.** I can reinstall later when an opportunity comes along - even reinstall OS if that's what it takes. There's always the possibility to reopen the issue later if nothing helps.

**Comment by cartermp** [comment]
Sounds good!

**Comment by BentTranberg** [comment]
Necro-note

I discovered accidentally late in 2019 what the source of this problem probably was all the time, and still is.

Certain monitors simply aren't able to show me the difference between the grayed out text and normal text. I discovered this when dragging editor windows back and forth between two monitors, where one of them clearly did show me the difference, and the other was not capable of that, at least not to my eyes.

I know I've had problems seeing grayed out text in F# way back in VS 2010 or 12 or 15 or something, and back then I believe it was possible to reconfigure the gray-out color. I could not find such a color in VS 2019 when investigating this issue. It's possible, but not certain, I should have been more persistent in that search.
